xmppserver/src/main/webapp/session-details.jsp

```diff
@@ -356,7 +356,33 @@
             Invalid session/connection
                 <% }
             } %>
+        </td>
     </tr>
+    <% 
+        try { 
+            if(currentSess.getSoftwareVersion()!= null ){ %>
+                <tr>
+                    <td class="c1">
+                        <fmt:message key="session.details.software_version"/>
+                    </td>
+                </tr>
+                <% 
+                for (Map.Entry<String, String> entry : currentSess.getSoftwareVersion().entrySet()){ %>
+                    <tr>
```

Comment:
The order of the entrySet is undefined, so different sessions may show the same information in a different order, which would be counter-intuitive.

Reply:
I don't think it's to much of a problem, but maybe we can order them alphabetically?